java/algorithms/Tsp-MemSat01/Main.java

```diff
@@ -0,0 +1,123 @@
+import org.sosy_lab.sv_benchmarks.Verifier;
```

Comment:
From my point of view, this file is missing an assert statement. What is the property, that should be proved here?